app/controllers/api/v1/planes_controller.rb

```diff
@@ -4,6 +4,22 @@ def index
     render json: PlaneSerializer.new(@planes).serializable_hash.to_json
   end
 
+  def create
+    unless admin?
+      return render json: { error: "You don't have enough permission to create new plane" },
+                    status: :unprocessable_entity
+    end
```

Comment:
Please kindly change the status into unauthorized so that frontend can know what's wrong with the code. I mean the status code should be 401 not 422.

Reply:
Optional:
if possible make a `function` for this and add `before_action` at the top of the class and specific the pages that is required. In that way we can use that function in whole class.

Reply:
Optional:
if possible make a `function` for this and add `before_action` at the top of the class and specific the pages that is required. In that way we can use that function in whole class.